static/app/components/nav/issueViews/issueViewNavItemContent.tsx

```diff
@@ -312,21 +312,24 @@ const StyledSecondaryNavItem = styled(SecondaryNav.Item)`
   position: relative;
   padding-right: ${space(0.5)};
 
-  :hover {
+  /* Hide the ellipsis menu if not hovered, or if it's not expanded  */
+  :not(:hover):not(:has([data-ellipsis-menu-trigger][aria-expanded='true'])) {
```

Comment:
Instead of checking for aria expanded you could check `focus-within`. That would do the same thing and also show the menu when the item has focus